LayoutTests/scrollbars/custom-scrollbar-scroller-style-change-expected.txt

```diff
@@ -0,0 +1,9 @@
+layer at (0,0) size 800x600
```

Comment:
Does this render tree dump sufficiently test the bug? If so, there's no need to add the PNG file. If not, it should be converted to a reference test.

Reply:
The render tree dump would be enough for this issue (both wk1 & 2 verified). Could you please further explain the reference test since I couldn't find any materials about this.

Reply:
A reference test uses two html files: `foo.html`, and `foo-expected.html` which are expected to render identically. You construct `foo-expected.html` to not exercise the buggy code, but give the same pixel results.